### DISCUSSION TITLE
Welcome to nodezator Discussions!

### DISCUSSION THREAD

**Discussion by KennedyRichard** [IndiePython/nodezator#4] [Announcements]
# 👋 Welcome to Nodezator Discussions!

Nodezator discussions will be the place for:

- questions
-  ideas
-  suggestions
-  constructive criticism

All related to the Nodezator app and its usage. You can also think about it like an **online forum dedicated to Nodezator**. Just ask away, be polite and all should go well.

:exclamation:   You are not required to, but if it is okay with you, please, make a brief comment below introducing yourself, the work or hobby that brought you here and your expectations for this community. This will help other members understand your background, making interaction easier and even find people with similar interests.

**Comment by KennedyRichard** [comment]
I'm Kennedy Guerra, 31. You might already know, but I created the Nodezator app and released it to the public domain, so it is actually everybody's now.

I programmed as a hobby since my teenage years but only got serious recently. While programming as a hobby, I learned a bit of Javascript, PHP and Python. I even made a small CMS in PHP to help me update my website from the web, instead of having to use ssh. Then I also learned Python and kept making small scripts. I have a business administration degree from 2015, but I do not work on the field.

Since 2016, I've been taking Python seriously and practicing everyday. I taught myself by reading technical books and making personal projects, one of which became Nodezator. I still have a lot to learn and I hope we can learn together while using and improving the Nodezator app.

**Comment by WillAdams** [comment]
William Adams, 56. On my 3rd, maybe 4th career (depending on how you count). Traditionally trained as a graphic designer/typographer, my current interest is automating a CNC machine and developing new designs and styles of joinery --- current tools include BlockSCAD, GraphSCAD, OpenSCAD, and I did a paper on using METAPOST in lualatex as well.

Hoping to work up a way to have Nodezator allow one to create designs which are simultaneously written out as OpenSCAD files (for previewing in 3D) and as G-code (for cutting out using a CNC machine)
